tools/code-owners-parser/ci.yml

```diff
@@ -27,9 +27,11 @@ extends:
     ToolDirectory: tools/code-owners-parser
     DotNetCoreVersion: 5.0.301
     TestPostSteps:
-      # This is a simple test for placeholder. Will replace the test with get-codeowners.ps1 in future.
-      - pwsh: |
-          $output = dotnet run --project "tools\code-owners-parser\Azure.Sdk.Tools.RetrieveCodeOwners\Azure.Sdk.Tools.RetrieveCodeOwners.csproj" `
-                      --code-owner-file-path "$(Build.SourcesDirectory)/tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners.Tests/CODEOWNERS" --target-directory "sdk"
-          $output | ConvertFrom-Json
-        displayName: Test on code owner tool output
\ No newline at end of file
+      - task: PowerShell@2
+        displayName: Test on code owner tool output
+        inputs:
+          pwsh: true
+          filePath: eng/common/scripts/get-codeowners.ps1
```

Comment:
While this is calling the script it isn't validating anything other than we don't get an error. I'd suggest we add a Test function inside of the script and add a "-Test" parameter that causes those tests to happen. Similar to the script I pointed you for cspell that Daniel is doing.